.gitignore

```diff
@@ -0,0 +1 @@
+/out/idea/
```

Comment:
```suggestion
/out/
.idea/
```

These are two different folders. Why did you put out/idea? when idea folder isn't inside the out folder